app/src/main/java/chat/rocket/android/files/ui/FilesFragment.kt

```diff
@@ -86,6 +86,7 @@ class FilesFragment : Fragment(), FilesView {
                     }
                 })
             }
+            group_no_file.isVisible = dataSet.isEmpty()
```

Comment:
Good catch @divyanshub024!

Thank you very much!

Reply:
:+1: